sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -207,8 +211,7 @@ def fetch(self, qres, path=None, error_callback=None, **kwargs):
         for i, [url, qre] in enumerate(zip(urls, qres)):
             name = url.split('/')[-1]
 
-            # temporary fix !!! coz All QRBs have same start_time values
-            day = Time(qre['Time'].start.strftime('%Y-%m-%d')) + TimeDelta(i*u.day)
+            day = qre['Time'].start
```

Comment:
why bother with the intermediate variable, just put this on line 220?

Reply:
yeah good point - left over from before, I'l l fix it now